### ISSUE TITLE
order_by search parameter still doesn't work properly 🐛 

### ISSUE THREAD

**Issue by mounir-laity** [jikan-me/jikan-rest#414]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I get a list of anime ordered by their mal_id when requesting the following url : 
https://api.jikan.moe/v4/anime?sfw&order_by=popularity 

### Expected Behavior

Have a list of anime ordered by their popularity.

### Steps To Reproduce

You can check this url : https://api.jikan.moe/v4/anime?sfw&order_by=popularity 
Or you can run the following code : 
```
import requests
import json

url = "https://api.jikan.moe/v4/anime?sfw"

params = {"order_by": "popularity"}

response = requests.get(url, params)
pretty_json = json.loads(response.text)

with open("result.json", "w") as file:
    file.write(json.dumps(pretty_json, indent=2))
```

### Environment

```markdown
Used public API
OS : MacOS 13.4.1
Browser : Firefox 114.0.2
```

### Anything else?

This issue seems pretty similar to issue #387 

But it hasn't been resolved by commit #410

**Comment by pushrbx** [comment]
This will be fixed in #409

**Comment by irfan-dahir** [comment]
This is resolved
